### PR TITLE
Improve description and re-order quickstart (#1)

### PR DESCRIPTION
**Pull request by bolkedebruin**
I've (hopefully) improved the description and adjusted the order of the quickstart to fit the natural order of things a bit better.

cc @kaxil @ashb

### CODE REVIEW

**Review by tatiana — APPROVED** [review]
LGTM, thanks, @bolkedebruin !

**Comment by tatiana** [comment]
It may be worth having @jwitz @mikeshwe review this, so our documentation is consistent.

**Review by jwitz — APPROVED** [review]
Looks great!

File: README.md [hunk 1]
```diff
@@ -15,11 +15,7 @@
 [![CI](https://github.com/astronomer/astro-sdk/actions/workflows/ci.yaml/badge.svg)](https://github.com/astronomer/astro-sdk)
 [![codecov](https://codecov.io/gh/astronomer/astro-sdk/branch/main/graph/badge.svg?token=MI4SSE50Q6)](https://codecov.io/gh/astronomer/astro-sdk)
 
-**Astro Python SDK** allows for rapid and clean development of extract, transform, and load (ETL) workflows using Python.
-
-The SDK abstracts the boilerplate code required for communication between datasets and tasks, which helps DAG authors to achieve more with less code.
-
-It is powered by [Apache Airflow](https://airflow.apache.org) and maintained by [Astronomer](https://astronomer.io).
+**Astro Python SDK** is a Python SDK for rapid development of extract, transform, and load workflows in [Apache Airflow](https://airflow.apache.org/). It allows you to express your workflows as a set of data dependencies without having to worry about ordering and tasks. The Astro Python SDK maintained by [Astronomer](https://astronomer.io).
```
Comment (kaxil) [inline README.md:18]:
```suggestion
**Astro Python SDK** is a Python SDK for rapid development of extract, transform, and load workflows in [Apache Airflow](https://airflow.apache.org/). It allows you to express your workflows as a set of data dependencies without having to worry about ordering and tasks. The Astro Python SDK is maintained by [Astronomer](https://astronomer.io).
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**